scalanet/src/io/iohk/scalanet/peergroup/UDPPeerGroup.scala

```diff
@@ -315,6 +323,10 @@ class UDPPeerGroup[M](val config: Config)(
 
 object UDPPeerGroup {
 
+  def getMessageQueue[M]() = {
```

Comment:
```suggestion
  def getMessageQueue[M] = {
```
Since there's no state change by calling this method.